privaterelay/settings.py

```diff
@@ -110,6 +110,7 @@
 
 RELAY_FROM_ADDRESS = config('RELAY_FROM_ADDRESS', None)
 SITE_ORIGIN = config('SITE_ORIGIN', None)
+GOOGLE_ANALYTICS_ID = config('GOOGLE_ANALYTICS_ID', None)
```

Comment:
Do we need `GOOGLE_ANALYTICS_ID` to add this to `.env-dist` ?

Reply:
Added.